src/main/java/frc/robot/commands/powerCellScoringCommands/IndexController.java

```diff
@@ -0,0 +1,5 @@
+package frc.robot.commands.powerCellScoringCommands;
```

Comment:
We already have an IndexerController class, we shouldn't need an IndexController.  I think I accidentally created this.